collector/sys_user_summary.go

```diff
@@ -0,0 +1,163 @@
+// Copyright 2022 Eduardo J. Ortega U.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package collector
+
+import (
+	"context"
+	"database/sql"
+	"github.com/go-kit/log"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const sysUserSummaryQuery = `
+	SELECT
+		user,
+		statements,
+		statement_latency,
+		statement_avg_latency,
```

Comment:
We probably don't need this as it can be derived from statements/statement_latency columns.

Reply:
Removed.